docs/orchestrating-elastic-stack-applications/orchestrating-elastic-stack-applications.asciidoc

```diff
@@ -12,7 +12,8 @@ endif::[]
 - <<{p}-elasticsearch-specification>>
 - <<{p}-kibana>>
 - <<{p}-apm-server>>
-- <<{p}-elastic-agent>>
+- <<{p}-elastic-agent-standalone>>
```

Comment:
I think standalone and fleet should have separate sections. There are too many differences (I feel) to combine them into single "Quickstart", "Configurations" and "Configuration examples". Another option I considered was to have another level in the menu, but I don't think we did that before so I didn't want to create a precedent. @alaudazzi, can you weigh in here?

Reply:
I agree that it's good to have two different sections given all the differences between the two but I'd appreciate a top section to clearly explain the differences between the 3 modes. I feel it can be a bit confusing.

Something like that:

The custom resource `Agent` allows you to deploy Fleet Server or the Elastic Agent in standalone or Fleet-managed mode, depending on the values of the `spec.mode` and `spec.fleetServerEnabled` elements:
- the Elastic Agent standalone, which is the default without setting `mode` and `fleetServerEnabled`
- the Fleet-managed Elastic Agent, with `mode: fleet` and without `fleetServerEnabled` which defaults to `false`
- the Fleet Server, with `mode: fleet` and `fleetServerEnabled: true`

It would be a repetition but a piece of yaml with callouts could help to identify:

```
kind: Agent
metadata:
  name: standalone-agent-quickstart
spec:
  version: 7.14.0
  elasticsearchRefs:
  - name: es-quickstart
---
kind: Agent
metadata:
  name: fleet-managed-agent-quickstart
spec:
  version: 7.14.0
  mode: fleet <1>
  fleetServerRef: <2>
    name: fleet-server-quickstart
  kibanaRef: <3>
    name: kb-quickstart
  elasticsearchRefs:
  - name: es-quickstart
---
kind: Agent
metadata:
  name: fleet-server-quickstart
spec:
  version: 7.14.0
  mode: fleet <4>
  fleetServerEnabled: true <4>
  kibanaRef: <3>
    name: kb-quickstart
  elasticsearchRefs:
  - name: es-quickstart
```

<1> Set `mode` to `fleet` to deploy the Agent in Fleet-managed mode, the opposite of standalone, see section <xxx>.
<2> Set `fleetServerRef` for the  Fleet-managed Elastic Agent, see section <yyy>.
<3> Set `kibanaRef` for Fleet-managed Elastic Agent or Fleet Server, see section <zzz>.
<4> Set `mode` to `fleet`  and `fleetServerEnabled` to true to deploy the Fleet Server, see section <www>.

Reply:
> I think standalone and fleet should have separate sections.

Sorry, I was not clear. I'm fine with having dedicated sections, my thinking was to avoid to add a new item in an already long list of `Run xxx on ECK`. The idea would be to have those sections under a `Run Fleet on ECK`, I think this product deserves it, but not strong opinion though..